src/Integration/Telemetry/TelemetryPayloadCreator.cs

```diff
@@ -84,23 +90,18 @@ internal static bool IsSonarCloud(Uri sonarqubeUri)
                    sonarqubeUri.Equals("https://www.sonarcloud.io/");
         }
 
-        internal static TelemetryPayload CreatePayload(TelemetryData telemetryData, DateTimeOffset now,
-            BindingConfiguration bindingConfiguration, IVsVersion vsVersion)
+        public TelemetryPayload Create(TelemetryData telemetryData)
```

Comment:
United the static method with the interface one, as nothing references the static method anymore (fixed the tests)